modules/enable-offline-mode-roxctl.adoc

```diff
@@ -10,21 +10,21 @@ You can enable offline mode when you are installing {product-title} by using the
 
 .Procedure
 
-. If you are using a registry other than the default internet-connected registry (`stackrox.io`), provide the locations where you have pushed the {product-title} images when answering the `image to use` prompts:
+. If you are using a registry other than the default internet-connected registry (`registry.redhat.io`), provide the locations where you have pushed the {product-title} images when answering the `image to use` prompts:
 +
 [source,terminal,subs=attributes+]
 ----
-Enter main image to use (default: "registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}"): <your_registry>/stackrox/main:{rhacs-version}
+Enter main image to use (if unset, the default will be used): <your_registry>/rhacs-main-rhel8:{rhacs-version}
 ----
```

Comment:
@misberner These new image names like `rhacs-main-rhel8`, `rhacs-scanner-db-rhel8` will be relevant for images mirrored from `registry.redhat.io` (i.e. `rhacs` flavor).
For images mirrored from `stackrox.io`, image names must be like `main` and `scanner-db`.
How do you think, should we write another section or footnotes for customers that mirror from `stackrox.io`?

Reply:
No, we want to get everyone off of stackrox.io. If they need to change anything about the mirroring config, the change we want them to make is to change the registry to registry.redhat.io.